simulation/replication/testdata/replication_simulation_reset.yaml

```diff
@@ -14,13 +14,9 @@ primaryCluster: "cluster0"
 
 domains:
   test-domain:
-    name: test-domain
-    activeClusters:
-    - cluster0
+    activeClusterName: cluster0
   test-domain-2:
-    name: test-domain-2
-    activeClusters:
-    - cluster1
+    activeClusterName: cluster0
```

Comment:
should it be `cluster1`?

Reply:
previously primary cluster name was used as active cluster name so both of these domains were active on cluster0. If @fimanishi 's intent was different then can be addressed in follow up PR.